python/lsst/ci/prepare.py

```diff
@@ -264,6 +281,28 @@ def fetch(self, product):
 
         """
 
+        for i in range(self.retries - 1):
```

Comment:
I don't understand this range. The default value is 1 retry, therefore I would expect that to run the fetch 1 or 2 times. If retries is 1 the code above won't run the fetch once. I would expect this to be `retries + 1`.

Reply:
Good point, the parameter should be named `tries` instead of `retries`.

Reply:
it should try at least once though?

Reply:
The loop doesn't need to run at all when there is going to be only one attempt, there is a final `_fetch` call outside of the loop so that exceptions will propagate when giving up.